templates/server.erb

```diff
@@ -0,0 +1,10 @@
+server <%= @name %> {
+  bogus <%= @bogus ? 'yes' : 'no' %>;
+  edns <%= @edns ? 'yes' : 'no' %>;
+<%- if @key -%>
+  keys <%= @key %>;
```

Comment:
There can be multiple `keys` values. I'd like to see support for `@key` to be either a single string value or an array of string values. There are some examples of similar parameter handling in other templates.